cibyl/orchestrator.py

```diff
@@ -131,7 +132,7 @@ def create_ci_environments(self) -> None:
 
                 self.environments.append(environment)
             self.set_system_api()
-        except AttributeError as exception:
+        except (AttributeError, TypeError) as exception:
```

Comment:
Would you really get a TypeError here? I do not see the reason why.

Reply:
That's actually why I added it :D